packages/framework/client-logger/external-telemetry/package.json

```diff
@@ -0,0 +1,76 @@
+{
+	"name": "@fluidframework/external-telemetry",
+	"version": "2.0.0-rc.3.0.0",
+	"description": "External Fluid telemetry types and classes for both creating consuming said telemetry",
+	"homepage": "https://fluidframework.com",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/microsoft/FluidFramework.git",
+		"directory": "packages/framework/client-logger/external-telemetry"
+	},
+	"license": "MIT",
+	"author": "Microsoft and contributors",
+	"main": "dist/index.js",
+	"module": "lib/index.mjs",
+	"types": "dist/index.d.ts",
+	"scripts": {
+		"api": "fluid-build . --task api",
+		"api-extractor:commonjs": "api-extractor run --local",
+		"api-extractor:esnext": "api-extractor run --config ./api-extractor-esm.json",
+		"build": "fluid-build . --task build",
+		"build:commonjs": "fluid-build . --task commonjs",
+		"build:compile": "fluid-build . --task compile",
+		"build:docs": "fluid-build . --task api",
+		"build:esnext": "tsc-multi --config ../../../../common/build/build-common/tsc-multi.esm.json",
+		"check:are-the-types-wrong": "attw --pack . --entrypoints .",
+		"check:release-tags": "api-extractor run --local --config ./api-extractor-lint.json",
+		"ci:build:docs": "api-extractor run",
+		"clean": "rimraf --glob _api-extractor-temp coverage dist lib nyc \"**/*.tsbuildinfo\" \"**/*.build.log\"",
+		"test": "npm run test:mocha",
+		"test:mocha": "mocha --recursive \"dist/test/**/*.spec.*js\" -r node_modules/@fluid-internal/mocha-test-setup",
+		"tsc": "tsc",
+		"typetests:gen": "fluid-type-test-generator",
+		"typetests:prepare": "flub typetests --dir . --reset --previous --normalize"
+	},
+	"dependencies": {
+		"@microsoft/applicationinsights-web": "^2.8.11"
+	},
+	"devDependencies": {
+		"@arethetypeswrong/cli": "^0.13.3",
+		"@fluid-internal/client-utils": "workspace:2.0.0-rc.3.0.0",
+		"@fluid-internal/mocha-test-setup": "workspace:2.0.0-rc.3.0.0",
+		"@fluid-tools/build-cli": "^0.34.0",
+		"@fluidframework/build-tools": "^0.34.0",
+		"@fluidframework/container-definitions": "workspace:~",
+		"@fluidframework/driver-definitions": "workspace:2.0.0-rc.3.0.0",
+		"@microsoft/api-extractor": "^7.42.3",
+		"@types/mocha": "^9.1.1",
+		"eslint": "~8.55.0",
+		"mocha": "^10.2.0",
+		"rimraf": "^4.4.0",
+		"sinon": "^17.0.1",
+		"tsc-multi": "^1.1.0",
+		"tslib": "^1.10.0",
+		"typescript": "~5.1.6"
+	},
+	"fluidBuild": {
+		"tasks": {
+			"build:docs": {
+				"dependsOn": [
+					"...",
+					"api-extractor:commonjs",
+					"api-extractor:esnext"
+				],
+				"script": false
+			},
+			"tsc": [
+				"...",
+				"typetests:gen"
+			]
+		}
+	},
+	"typeValidation": {
+		"broken": {},
+		"disabled": true
+	}
```

Comment:
Was this because it needs to be disabled until we have a previous version?

Reply:
That is correct.

Reply:
Yep